src/plots/cartesian/transition_axes.js

```diff
@@ -17,6 +17,8 @@ var Drawing = require('../../components/drawing');
 var Axes = require('./axes');
 var axisRegex = /((x|y)([2-9]|[1-9][0-9]+)?)axis$/;
 
+var LAST_TRANSLATION_RE = /translate\([^)]*\)\s*$/;
```

Comment:
Can you just use [`Drawing.getTranslate`](https://github.com/plotly/plotly.js/blob/master/src/components/drawing/index.js#L588) instead?